.github/workflows/acceptance-tests-runner.yml

```diff
@@ -382,6 +382,38 @@ jobs:
             ./internal/service/advancedclustertpf
         run: make testacc
 
+  advanced_cluster_tpf_move:
+    needs: [ change-detection, get-provider-version ]
+    if: ${{ (needs.change-detection.outputs.advanced_cluster_tpf == 'true' || inputs.test_group == 'advanced_cluster_tpf') && inputs.terraform_version == '1.10.x' }}
```

Comment:
run only if testing the latest version, for instance don't run if running compatibility matrix each month